scl/loadbalancer/gen-loadbalancer.sh

```diff
@@ -1,4 +1,25 @@
 #!/bin/sh
+#############################################################################
+# Copyright (c) 2017 Balabit
```

Comment:
I can, but why?
it does two different things on two different modules.

Reply:
Sorry I meant not those two commits into one :)

You have currently 4 commits, from which 2 of them just for style check and copyright.
It would be nice to squash this modification into the original two commits (those are well separated).

The **confgen: fixed coding style error** and **confgen: enable passing arguments to confgen external program as env** into one, also the **loadbalancer: added missing copyright headers** and  **loadbalancer: added network load balancer destination** into one.

I hope it make more sense like this :)

Reply:
fair, i will do it :)